### ISSUE TITLE
[Word Desktop]: Some improper announcement by NVDA while selecting an Image/Illustration.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8278]
### Steps to reproduce:
1. Open Sample doc
2. Use Down arrow and navigate to the left or right of some image.
3. Use Shift + left/right arrow for the image selection.
4. Observe the NVDA announcement. NVDA is announcing "Slash Selected".
5. Same issue is happening while any of the illustration selection (Shape, SmartArt, Chart)

### Expected behavior:
NVDA should announce the image selection as per expectation.

### Actual behavior:
NVDA is not announcing image selection properly. It is announcing "Slash selected" when selection using Shift + Arrow key. Automatic selection with Down arrow is working properly.

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by derekriemer** [comment]
I can reproduce this one. It's happened for a long time, and I think Jaws did it too. You can get it to happen in word with UIA to.

**Comment by WiproPSAccTest** [comment]
![image selection](https://user-images.githubusercontent.com/22956130/40101592-9092ca7e-5905-11e8-803a-b42b54c813f8.png)

**Comment by Adriani90** [comment]
Duplicate of #6273. Closing.
